src/config.py

```diff
@@ -1,25 +0,0 @@
-import os
```

Comment:
I did it this way to try to keep the test and prod as similar as possible. This way they share much of the config directly. That's also why I added a snapshot test for prod config, so that we can use this shared file with confidence that we're not changing prod config accidentally.

Reply:
Will the new Config class now suffice for this purpose too? https://github.com/quokkateam/quokka-api/pull/29/files#diff-4d8e079a99db68b5feca3d7216a49b10

Reply:
Using inheritance to share the config